java/src/jmri/jmrit/beantable/LRouteTableAction.java

```diff
@@ -1132,7 +1132,7 @@ public void windowClosing(java.awt.event.WindowEvent e) {
                     // remind to save, if Route was created or edited
                     if (routeDirty) {
                         InstanceManager.getDefault(jmri.UserPreferencesManager.class).
-                                showInfoMessage("Reminder", "Remember to save your LRoute information.", getClassName(), "remindSaveRoute");
+                                showInfoMessage(Bundle.getMessage("ReminderTitle"), Bundle.getMessage("ReminderSaveString", "LRoute"), getClassName(), "remindSaveRoute");
```

Comment:
This needs a ```// NOI18N``` marker to indicate that these String literals do not need translation.